data-collection/deploy/module-securityhub.yaml

```diff
@@ -0,0 +1,452 @@
+    Type: "AWS::IAM::Role"
+    DeletionPolicy: "Delete"
+    Properties:
+      Path: "/service-role/"
+      MaxSessionDuration: 3600
+      RoleName: !Sub "${ResourcePrefix}${CFDataName}-EBInvoke_Firehose"
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+        - Action: "sts:AssumeRole"
+          Effect: "Allow"
+          Principal:
+            Service: "events.amazonaws.com"
+      Metadata:
+        cfn_nag:
+          rules_to_suppress:
+            - id: W28 # Resource found with an explicit name, this disallows updates that require replacement of this resource
+              reason: "Need explicit name to identify role actions"
+
+    KinesisFirehoseRole:
+      Type: "AWS::IAM::Role"
+      Properties:
+        Path: "/"
+        MaxSessionDuration: 3600
+        RoleName: "${ResourcePrefix}${CFDataName}-KinesisFireHoseRole"
+        Description: "Allows Kinesis Firehose to transform and deliver data to your\
+          \ destinations using CloudWatch Logs, Lambda, and S3 on your behalf."
+        Policies:
+          - PolicyDocument:
+              Version: "2012-10-17"
+              Statement:
+              - Resource:
+                - !Sub "arn:${AWS::Partition}:s3:::${CodeBucket}"
+                - !Sub "arn:${AWS::Partition}:s3:::${CodeBucket}*"
+                - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:%FIREHOSE_POLICY_TEMPLATE_PLACEHOLDER%:log-stream:*"
+                - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/kinesisfirehose/PUT-S3-SVAnj:log-stream:*"
+                - !Sub "arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:${LambdaFunction}"
+                Action:
+                - "s3:PutObject"
+                - "s3:GetObject"
+                - "s3:ListBucketMultipartUploads"
+                - "s3:AbortMultipartUpload"
+                - "lambda:InvokeFunction"
+                - "s3:ListBucket"
+                - "lambda:GetFunctionConfiguration"
+                - "s3:GetBucketLocation"
+                - "logs:PutLogEvents"
+                Effect: "Allow"
+                Sid: "VisualEditor0"
+            PolicyName: "${CFDataName}-detail-KinesisDataStream"
+        AssumeRolePolicyDocument:
+          Version: "2012-10-17"
+          Statement:
+          - Action: "sts:AssumeRole"
+            Effect: "Allow"
+            Principal:
+              Service: "firehose.amazonaws.com"
+      Metadata:
+        cfn_nag:
+          rules_to_suppress:
+            - id: W28 # Resource found with an explicit name, this disallows updates that require replacement of this resource
+              reason: "Need explicit name to identify role actions"
+
+  EventRuleSechubKinesisIntegration:
+    Type: "AWS::Events::Rule"
+    Properties:
+      EventPattern:
+        detail-type:
+        - "Security Hub Findings - Imported"
+        source:
+        - "aws.securityhub"
```

Comment:
does this means that the data collection account should be a delegated admin for SH?

Reply:
It can or cannot. Account where Data collection is enabled along with SH. It can also be a delegated admin account where Data collection is running as well. Idea is, that the customer should enable at the main account and then can enable QuickSight access to other admins

Reply:
Can you add a comment about this to Readme? That Data Collection account must be an Admin or Delegated admin of SH?